index.html

```diff
@@ -2,9 +2,174 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Een blok positioneren</title>
+    <link rel="stylesheet" href="./styles.css">
 </head>
 <body>
+<header class="header__outer-container">
```

Comment:
Als je de header een id attribuut geeft, kunnen we met de links naar de individuele stukken linken. 
`<header id="header" en class enz>` 

in de links:
`<a href="#header">`